migrations/2021_02_12_134612_create_webhook_notification_messages_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWebhookNotificationMessagesTable extends Migration
+{
+    public function up(): void
```

Comment:
Migrations should be publishable in the ServiceProvider.

Reply:
That's done in the `LaravelWebhookNotificationServiceProvider`.